source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -7,17 +7,39 @@
 
 #include "common/common/assert.h"
 
+#include "extensions/filters/network/redis_proxy/config.h"
+#include "extensions/filters/network/well_known_names.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace NetworkFilters {
 namespace RedisProxy {
 namespace ConnPool {
 
+namespace {
+
+Common::Redis::Client::DoNothingPoolCallbacks null_pool_callbacks;
+
+Common::Redis::RespValue makeAuthCommand(const std::string& password) {
```

Comment:
Any difference between this and `Extensions::NetworkFilters::Common::Redis::RespValue makeAuthCommand(const std::string& password)` in `source/extensions/clusters/redis/redis_cluster.cc`?

Reply:
no ... I'll create a single instance in Extensions::NetworkFilters::Common::Redis::Utility namespace (and add a new utility.cc and utility.h to extensions/filters/network/common/redis)...